adapter/stackdriver/BUILD

```diff
@@ -0,0 +1,36 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = glob(
```

Comment:
I wrote this before we decided that. I forgot I even used globs here, since I didn't need to update the BUILD file when I added go files ;).